README.md

```diff
@@ -5,6 +5,8 @@
 
 FirstGiving Ruby Client Open Source.
 
+EightBit Studios Fork to include the 'CARDONFILE' POST call
```

Comment:
Do you intend to use this as a fork, or merge to the main repo? If no, consider removing this line?